.github/workflows/build.yaml

```diff
@@ -0,0 +1,38 @@
+name: "build"
+on: [push, pull_request]
+env:
+  TRIVY_VERSION: 0.43.1
+  COPA_VERSION: 0.3.0
+permissions: write-all
+jobs:
+  build:
+    name: build
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Setup BATS
+        uses: mig4/setup-bats@v1
+        with:
+          bats-version: 1.7.0
+
+      - name: Check out code
+        uses: actions/checkout@v1
+
+      - name: Install Trivy
+        run: |
+            curl -sfL https://raw.githubusercontent.com/aquasecurity/trivy/main/contrib/install.sh | sh -s -- -b /usr/local/bin v${{ env.TRIVY_VERSION }}
```

Comment:
can we download the binary directly or use the trivy action? we should avoid using `curl | bash` since the url can change at any time

Reply:
I don't think we can use the action since that is to run a trivy command, but can change it to install the binary directly